tests/ctst/common/common.ts

```diff
@@ -55,10 +55,41 @@ export async function cleanS3Bucket(
     await S3.deleteBucket(world.getCommandParameters());
 }
 
+/**
+ * @param {Zenko} this world object
+ * @param {string} objectName object name
+ * @returns {string} the object name based on the backend flakyness
+ */
+function getObjectNameWithBackendFlakyness(this: Zenko, objectName: string) {
+    let objectNameFinal;
+    const backendFlakynessRetryNumber = this.getSaved<string>('backendFlakynessRetryNumber');
```

Comment:
shouldn't it a number that w increment if we pass a retry number > 1?

Reply:
I am not sure I understand why we should increment it ?

Reply:
From what I understand you set `retryNumber` as a test parameter, which is 1 here, that you eventually use as the object name to work with. If you set a number > 1, we never retry twice, and the object name will have `-2` in its name... Maybe I misunderstood but to me what you are trying to implement is a way to retry a scenario step and the ability to track those objects from the logs, right?
That's why I think it should be incremented every time we retry, otherwise there is no need to have a test parameter if the value is static